item-impl/src/test/scala/com/example/auction/item/impl/ItemServiceImplIntegrationTest.scala

```diff
@@ -5,18 +5,18 @@ import java.util.UUID
 
 import akka.stream.scaladsl.Sink
 import com.example.auction.item.api
-import com.example.auction.item.api.{ItemService, ItemSummary}
+import com.example.auction.item.api.{ ItemService, ItemSummary }
```

Comment:
Maybe we should set up scalariform on this project to avoid formatting churn... I'll raise an issue.